### DISCUSSION TITLE
Where can I find plugins folks have made for Endless Sky?

### DISCUSSION THREAD

**Discussion by ValleyOfTheMariner** [endless-sky/endless-sky#7259] [General]
Hey, folks!

A friend and I have made some games together in the past, and we’ve been looking into what it would take to make an EV clone with a new storyworld.

After researching the space trading games that were out there to see if any would be good starting points for a mod or plugin, it seems obvious that Endless Sky is our best bet - better than starting from a different base game, and also better than reinventing the wheel on our own. Endless Sky preserves what I loved about EV while fixing a lot of its limitations.

Having now put in about 50 hrs in the base game, I was curious where I could find some existing plugins/mods to play as we research this possibility.

Thanks for making this cool game!

**Comment by quyykk** [comment]
We have a (pretty much) official list [on our website](https://endless-sky.github.io/plugins.html). However, there are many more plugins that exist. There's also [an archive](https://github.com/Hecter94/EndlessSky-PluginArchive) of pretty much every plugin.

**Reply by ValleyOfTheMariner** [reply]
Thank you! Will check these out.
